src/tiledb/cloud/vector_search/file_ingestion.py

```diff
@@ -0,0 +1,354 @@
+from typing import Dict, List, Mapping, Optional, Sequence
+
+from tiledb.cloud import dag
+from tiledb.cloud.utilities import as_batch
+
+
+def ingest_files_dag(
+    file_dir_uri: str,
+    index_uri: str,
+    file_name: Optional[str] = None,
+    acn: Optional[str] = None,
+    config=None,
+    environment_variables: Mapping[str, str] = {},
+    namespace: Optional[str] = None,
+    verbose: bool = False,
+    trace_id: Optional[str] = None,
+    # Index creation params
+    create_index: bool = False,
+    index_type: str = "IVF_FLAT",
+    index_creation_kwargs: Dict = {},
+    # DirectoryTextReader params
+    include: str = "*",
+    exclude: Sequence[str] = ["[.]*", "*/[.]*"],
```

Comment:
This is simpler; you can replace the list with a tuple; i.e., `("[.]*", "*/[.]*")`.

Reply:
Applied